src/cdk/wizards/previewStateMachineCDKWizard.ts

```diff
@@ -89,50 +102,92 @@ export default class PreviewStateMachineCDKWizard extends MultiStepWizard<Previe
         })
 
         this.cdkApplication = picker.verifySinglePickerOutput<CdkAppLocationPickItem>(choices)
-        return this.cdkApplication ? wizardContinue(this.TEMPLATE_FORMAT_ACTION) : WIZARD_GOBACK
+        return this.cdkApplication ? wizardContinue(this.SELECT_APPLICATION_ACTION) : WIZARD_GOBACK
     }
 
-    private readonly TEMPLATE_FORMAT_ACTION: WizardStep = async () => {
-
+    private readonly SELECT_APPLICATION_ACTION: WizardStep = async () => {
         const appLocation = this.cdkApplication ? this.cdkApplication.cdkApplocation : undefined
+
+        if (!appLocation) return WIZARD_GOBACK
+
         const appNode = new AppNode(appLocation!)
         const constructNodes = await appNode.getChildren()
-        //filter placeholder nodes
-        //constructNodes = constructNodes.filter(i => i.contextValue === 'awsCdkConstructNode' || i.contextValue === 'awsCdkStateMachineNode' )
+        const TOP_LEVEL_NODES: TopLevelNodePickItem[] = []
```

Comment:
Can keep the variable name in lowercase. Similar comment about assigning as the result of the map function. Can also rename the variable to something more descriptive.